### DISCUSSION TITLE
Individual chip colors

### DISCUSSION THREAD

**Discussion by mecha** [mantinedev/mantine#1681] [Feature requests]
### Usage example, including component/hook API

Allow individual chips in a `Chip` context to have different colors.

The `Chips` component currently overrides the `color` prop for its children, even if no color is specified. This overrides the `color` prop on the individual chips.

[Code Sandbox](https://codesandbox.io/s/happy-shaw-5kjhr3?file=/src/App.js)

### Possible implementation – describe how the feature can be implemented

In the [`React.clone()` call in the `Chips` component](https://github.com/mantinedev/mantine/blob/master/src/mantine-core/src/components/Chips/Chips.tsx#L92), the `color` prop can be defaulted to the child's `color` prop if the parent's color is undefined.

### Do you want to contribute to this feature and create a pull request?

Yes. Actually, it's [already finished](https://github.com/mecha/mantine/tree/feat/Chips/retain-child-colors). Do I just go ahead and submit it?

**Comment by rtivital** [comment]
It is already fixed in 5.0, no need to submit a PR

**Reply by mecha** [reply]
Fantastic.
